docs/docs/guides/web3_providers_guide/eip6963.md

```diff
@@ -40,8 +40,13 @@ Following code snippet demonstrates usage of `requestEIP6963Providers()` functio
 
 import { Web3 } from 'web3';
 
-const providers = Web3.requestEIP6963Providers();
+window.addEventListener('web3:providersMapUpdated', (event) => {
```

Comment:
I would maybe hide this behind something like `web3.onNewProviderDiscovered((provider) => {})`